src/import.js/importlist.js

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react'
+import Accordion from '@mui/material/Accordion'
+import AccordionSummary from '@mui/material/AccordionSummary'
+import AccordionDetails from '@mui/material/AccordionDetails'
+import Typography from '@mui/material/Typography'
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore'
+
+function ImportList(props) {
```

Comment:
you can extract props to omit getting props.title 

for instance
function ImportList({title, body}) {}